spec/models/default_bill_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe DefaultBill do
+  let!(:current_user) { create(:user) }
+
+  it 'is valid with valid params' do
+    expect(described_class.new(name: 'Default Bill', user: current_user)).to be_valid
+  end
+
+  it 'generates slug' do
+    default_bill = described_class.create(name: 'Default Bill', user: current_user)
+
+    expect(default_bill.slug).not_to be_nil
```

Comment:
Também daria pra fazer tipo:

```suggestion
    expect(default_bill.slug).to match(/default-bill-.*/)
```

Acho que isso funciona... Tem que testar 😆